torch/csrc/profiler/kineto_client_interface.cpp

```diff
@@ -18,16 +18,29 @@ class LibKinetoClient : public libkineto::ClientInterface {
     reportInputShapes_ = setupOpInputsCollection;
   }
 
+#ifdef USE_KINETO_UPDATED
+  void start(bool withStack) override {
+    ProfilerConfig cfg {
+      ProfilerState::KINETO_ONDEMAND,
+          /*report_input_shapes=*/reportInputShapes_,
+          /*profile_memory=*/false,
+          /*with_stack=*/withStack,
+#else
   void start() override {
     ProfilerConfig cfg{
         ProfilerState::KINETO_ONDEMAND,
-        reportInputShapes_,
-        false,
-        false,
-        false,
-        false};
+        /*report_input_shapes=*/reportInputShapes_,
+        /*profile_memory=*/false,
+        /*with_stack=*/false,
+#endif
+          /*with_flops=*/false,
+          /*with_modules=*/false
+    };
     std::set<ActivityType> activities{ActivityType::CPU};
-    auto scopes = {at::RecordScope::FUNCTION, at::RecordScope::USER_SCOPE};
+    std::unordered_set<at::RecordScope> scopes;
```

Comment:
I don't see why this was needed. Can't you just add an entry?

Reply:
There was a lint bug. This is just a workaround.